### ISSUE TITLE
Can I use this with videos?

### ISSUE THREAD

**Issue by fabd** [Eboreg/yt-dlp-replaygain#1]
Hey I hope it's ok to ask here. I have my own bash file to simplify some of the process of downloading some favourites from YT - however the audio normalization is definitely noticeable when I playback in Infuse on the AppleTV.

Just wondering if it's possible to use the plugin to normalize audio the same way the YT client does - but keeping the video as well?

Asking because I see your script uses `--extract-audio` but I'm not sure how it works if I let yt-dlp merge the video & audio.

So to summarize:

- can the plugin be used when keeping the video stream as well?
- would you have a command line example of how this would work?
 
Thanks
